app/api/feedback/route.tsx

```diff
@@ -62,13 +64,15 @@ export const POST = async (request: NextRequest) => {
       },
       { status: 200 }
     );
-  } catch (e) {
-    return NextResponse.json(
-      {
+  } catch (e: unknown) {
+    if (e instanceof Error) {
+      return NextResponse.json({
```

Comment:
Can you do `NextResponse.status(400).json....`? I think that's the syntax. Otherwise, it's losing the status code in the response.